src/styles/card.scss

```diff
@@ -0,0 +1,116 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$hover-title-color: #34568b;
+
+.card {
+  height: 408px;
+  max-width: 200px;
+  transition-duration: 0.6s;
+
+  &__img {
+    text-align: center;
+    margin-bottom: 37px;
+
+    img {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__title {
+    h2 {
+      font-family: Roboto, sans-serif;
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 500;
+      color: $main-accent;
+      margin-bottom: 4px;
+    }
+
+    p {
+      font-family: Roboto, sans-serif;
+      font-size: 10px;
+      font-weight: 400;
+      font-style: normal;
+      line-height: 14px;
+      color: $secondary;
+      margin-top: 4px;
+      margin-bottom: 16px;
+    }
+  }
+
+  &__rating {
+    height: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 12px;
+  }
+
+  &__review {
+    p {
+      font-family: Roboto, sans-serif;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $main-accent;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+    color: $secondary;
+
+    span {
+      font-family: Roboto, sans-serif;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $main-accent;
+    }
+  }
+
+  &__btn {
+    button {
+      border: 1px solid transparent;
+      font-family: Roboto, sans-serif;
+      border-radius: 5px;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      color: #fff;
+      background-color: $blue-accent;
+      width: 100%;
+      height: 40px;
+      margin-top: 4px;
+    }
+
+    button:hover {
+      border: 1px solid $blue-accent;
+      background-color: #fff;
+      color: $blue-accent;
+    }
+  }
+
+  &:hover {
+    cursor: pointer;
+    transform: scale(1.2);
+    transition-duration: 0.3s;
```

Comment:
The hover effect scales the card to 1.2 times its size. Ensure that this scaling does not affect the layout negatively, especially if multiple cards are displayed in a grid.